modules/k8s-nmstate-uninstall-operator.adoc

```diff
@@ -0,0 +1,212 @@
+// Module included in the following assemblies:
+//
+// networking/k8s_nmstate/k8s-nmstate-about-the-kubernetes-nmstate-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="k8s-nmstate-uninstall-operator_{context}"]
+= Uninstalling the Kubernetes NMState Operator
+
+You can use the {olm-first} to uninstall the Kubernetes NMState Operator, but by design {olm} does not delete any associated custom resource definitions (CRDs), custom resources (CRs), or API Services. 
+
+Before you uninstall the Kubernetes NMState Operator from the `Subcription` resource that {olm} uses, identify what Kubernetes NMState Operator resources to delete. This identification ensures that you can delete the resources without impacting your running cluster.
+
+.Procedure
+
+. Find `openshift-nmstate` namespace that exists in the `Subcription` resource that {olm} uses by running the following command:
++
+[source,terminal]
+----
+$ oc get --all-namespaces subscription
+----
++
+.Example output
+[source,terminal]
+----
+NAMESPACE           NAME                      	  PACKAGE                   	SOURCE                   	 CHANNEL
+openshift-nmstate   kubernetes-nmstate-operator   kubernetes-nmstate-operator   kubernetes-nmstate-catalog   stable
+...
+----
+
+. Unsubscribe the Kubernetes NMState Operator from the `Subcription` resource by running the following command:
++
+[source,terminal]
+----
+$ oc delete --namespace openshift-nmstate subscription kubernetes-nmstate-operator
+----
+
+. Find the `ClusterServiceVersion` (CSV) resource that associates with the Kubernetes NMState Operator by running the following command:
++
+[source,terminal]
+----
+$ oc get --namespace openshift-nmstate clusterserviceversion
+----
++
+.Example output that lists the CSV resource
+[source,terminal]
+----
+NAME                              	  DISPLAY                   	  VERSION   REPLACES   PHASE
+kubernetes-nmstate-operator.v4.18.0   Kubernetes NMState Operator   4.18.0           	   Succeeded
+----
+
+. Delete the CSV resource. After you delete the file, {olm} deletes certain resources, such as `RBAC`, that it created for the Operator.
++
+[source,terminal]
+----
+$ oc delete --namespace openshift-nmstate clusterserviceversion kubernetes-nmstate-operator.v4.18.0
+----
+
+. Find the `CatalogSource` resource for the Kubernetes NMState Operator that exists in the cluster and any pods by running the following commands:
++
+[source,terminal]
+----
+$ oc get --all-namespaces catalogsource | grep nmstate
+----
++
+.Example output
+[source,terminal]
+----
+openshift-marketplace   kubernetes-nmstate-catalog   Catalog for kubernetes-nmstate   grpc   knmstate-catalog   2d5h
+----
++
+[source,terminal]
```

Comment:
Does a need exist to determine a `CatalogSource` resource that exists in the pods?

Reply:
Nah, in fact, as I think about it, you can skip this step completely. Let's not mention CatalogSource here at all